modules/cli/src/test/scala/coursier/cli/ResolveTests.scala

```diff
@@ -342,9 +342,9 @@ class ResolveTests extends FlatSpec with BeforeAndAfterAll {
     val expectedOutput =
       """https://repo1.maven.org/maven2/com/github/alexarchambault/case-app_2.13/2.0.0-M9/case-app_2.13-2.0.0-M9.jar
         |https://repo1.maven.org/maven2/org/scala-lang/scala-library/2.13.0/scala-library-2.13.0.jar
-        |https://repo1.maven.org/maven2/com/chuusai/shapeless_2.13/2.3.3/shapeless_2.13-2.3.3.jar
         |https://repo1.maven.org/maven2/com/github/alexarchambault/case-app-annotations_2.13/2.0.0-M9/case-app-annotations_2.13-2.0.0-M9.jar
         |https://repo1.maven.org/maven2/com/github/alexarchambault/case-app-util_2.13/2.0.0-M9/case-app-util_2.13-2.0.0-M9.jar
+        |https://repo1.maven.org/maven2/com/chuusai/shapeless_2.13/2.3.3/shapeless_2.13-2.3.3.jar
```

Comment:
Wondering why this one changes 🤔

Reply:
Yeah, I kept myself from investing even more time diagnosing.  Thanks for having a look.

Reply:
I had missed that this corresponds to the `--candidate-urls` option. It gets its artifact list on its own, I pushed a commit for it to rely on `--classpath-order` if it's there.